NuWeather/plugin.py

```diff
@@ -225,7 +225,7 @@ def _format_distance(mi=None, km=None, speed=False):
             km = round(mi * 1.609, 1)
 
         if speed:
-            return _('%smph/%skph') % (mi, km)
+            return _('%smph/%sKm/h') % (mi, km)
```

Comment:
This should be `km/h` with a lowercase `k`

Reply:
Messed that up.